tests.py

```diff
@@ -920,8 +932,14 @@ def test_wrapInstance():
         finally:
             app.exit()
 
-    def test_implicit_wrapInstance():
-        """.wrapInstance doesn't need the `base` argument"""
+    def test_implicit_wrapInstance_for_base_types():
+        """Tests implicit `Foo` -> `Foo` cast of `Foo` pointer to `Foo` object,
+        produced by .wrapInstance when:
+
+        #. The `base` argument has a default value.
```

Comment:
Are these bullet lists? Would rather make them `1.` and `2.` in that case to keep focus on the code rather than the formatting.

Reply:
Yep it's just rst auto-enumeration, I've changed it to explicit.
If any of the elaboration is too much I'm happy to remove it.
Cheers

Reply:
Ooo, learnt something new. Thanks.